packages/ui/src/ui/UIFactory/default-ui-factory.tsx

```diff
@@ -0,0 +1,313 @@
+import React from 'react';
+import AppNavigationPageLayout from '../containers/AppNavigation/AppNavigationPageLayout';
+import {defaultAclApi} from '../utils/acl/external-acl-api';
+import {SupportComponentLazy} from '../containers/SupportComponent/lazy';
+import {YTUserSuggestLazy} from '../containers/UserSuggest/YTUserSuggestLazy';
+import {docsUrls} from '../constants/docsUrls';
+import {YTSubjectSuggestLazy} from '../containers/ACL/SubjectsControl/lazy';
+import {RoleActionsLazy} from '../containers/ACL';
+import OperationDetailMonitorLinks from '../pages/operations/OperationDetail/tabs/monitor/OperationDetailsMonitorLinks';
+import {PERMISSIONS_SETTINGS} from '../constants/acl';
+import {uiSettings} from '../config/ui-settings';
+import YT from '../config/yt-config';
+import {DefaultSubjectLinkLazy} from '../components/SubjectLink/lazy';
+import type {SubjectCardProps} from '../components/SubjectLink/SubjectLink';
+import {CUSTOM_QUERY_REQULT_TAB} from '../pages/query-tracker/QueryResultsVisualization';
+
+import {UIFactory} from './index';
+
+const experimentalPages: string[] = [];
+
+export const defaultUIFactory: UIFactory = {
```

Comment:
I don't quite understand the value of this transfer. Now I have one object here and an entry point to index

Reply:
It is better to review PRs by commits, then you will see more https://github.com/ytsaurus/ytsaurus-ui/pull/766/commits/49fc4a1cb34c83c8a9a1474e7b1de040127e738a